src/test/kotlin/CheckersModelTest.kt

```diff
@@ -0,0 +1,40 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertTrue
+
+class TestCanMove {
+
+    @Test
+    fun `Test Move of Ordinary Checker 1`() {
+        val m = CheckersModel()
+        assertTrue(m.canMove(BaseTurn(1, Pair(0, 0), Pair(1, 1))) == null,
+            "Moved into not empty square")
+        assertTrue(m.canMove(BaseTurn(-1, Pair(5, 0), Pair(4, 1))) == null,
```

Comment:
You may also use `to` here. But maybe `Pair` is better, I don't really know.

Reply:
I think that Pair() is better for pair of square coordinates